src/repoStore.ts

```diff
@@ -148,6 +148,14 @@ class RepoStore {
             repo.reIndex();
         });
     }
+
+    public reMultipleIndices(ids: string[]) {
+        _.forEach(this.repos, (repo: Repo) => {
```

Comment:
Convert `ids` to a `Set` so lookup is faster, instead of using `includes` every time

Reply:
> Convert `ids` to a `Set` so lookup is faster, instead of using `includes` every time

@ron-rookout Done :)